spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkWriteConf.java

```diff
@@ -484,42 +525,24 @@ private Map<String, String> deleteWriteProperties() {
 
     switch (deleteFormat) {
       case PARQUET:
-        setWritePropertyWithFallback(
```

Comment:
Instead of using `setWritePropertyWithFallback`, I added defaults to our delete configs.
There are tests in `TestCompressionSettings` and `TestSparkWriteConf` to verify the new logic.

Reply:
This seems fine to me, but was it related to the advisory partition size changes? It seems like it doesn't overlap.

Reply:
Correct, it was needed to compute the codec.